### PR TITLE
use User.roles for holding authz roles from the authn header

### PR DESCRIPTION
**Pull request by konstan**
Please don't merge it yet.  It still lacks full test with authn/z.

Connected to #1394

### CODE REVIEW

File: jar-persistence/src/main/java/com/sixsq/slipstream/persistence/User.java [hunk 1]
```diff
@@ -949,18 +950,18 @@ private void checkNoForbiddenRoles(String roles) throws ValidationException {
         if (roles == null || roles.isEmpty()) {
             return;
         }
-        for (String role : roles.split(",")) {
+        for (String role : roles.split(" ")) {
             String trimedUppercaseRole = role.trim().toUpperCase();
             if (FORBIDDEN_ROLES.contains(trimedUppercaseRole)) {
```
Comment (loomis) [inline jar-persistence/src/main/java/com/sixsq/slipstream/persistence/User.java]: This check will probably need to be removed as the ADMIN, USER, and ANON roles will probably already be present in the claims.

**Comment by loomis** [comment]
The last two commits remove the calls to the user object for roles.

File: jar-persistence/src/main/java/com/sixsq/slipstream/persistence/User.java (hunk 1 above)
Comment (konstan) [reply jar-persistence/src/main/java/com/sixsq/slipstream/persistence/User.java]: Yes, I also noticed that and wanted to fix it when running the full test with authn/z.  I'm preparing my dev env for the test at the moment.

**Comment by konstan** [comment]
@loomis This PR is ready for review.

**Pull request merged**